source/agora/node/Runner.d

```diff
@@ -320,7 +320,7 @@ private void runDNSServer_canThrow (in RegistryConfig config, NameRegistry regis
             auto pack = udp.recv(buffer, ppeer);
             auto query = deserializeFull!Message(pack);
             registry.answerQuestions(
-                query,
+                query, peer.toAddressString,
```

Comment:
Does it do a reverse DNS lookup or just spits out an IP ?

Reply:
Just string representation of the IP